hvac/api/auth_methods/token.py

```diff
@@ -555,6 +555,8 @@ def create_or_update_role(
         renewable=True,
         path_suffix=None,
         allowed_entity_aliases=None,
+        token_period=None,
+        token_explicit_max_ttl=None,
```

Comment:
Although it looks a little weird, let's move these new parameters down to the end, just to accommodate folks who might be setting parameters positionally.